rebar.config

```diff
@@ -5,7 +5,8 @@
 
 {deps, [
 	{cut, "1.0.3"},
-	{ppplib, "1.0.0"}
+	{ppplib, "1.0.0"},
+	{parse_trans, {git, "https://github.com/uwiger/parse_trans.git", {branch, "master"}}}
```

Comment:
Please keep an eye on the repo and/or ask for a tag instead of using master, just in case.